app/src/main/java/com/worldsnas/starplayer/model/Music.kt

```diff
@@ -1,10 +1,14 @@
 package com.worldsnas.starplayer.model
 
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
```

Comment:
afarin

Reply:
vali lotfan ye modele jadid besazi
esmesh ham mitonid `LocalMusic` ya chizi shabihesh bezarid
in aslan kare khobi nist ye object az payin tarin laye ta balatarin laye ro control kone